RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -27,14 +27,19 @@
 					   'keep *_hbheprereco_*_*', 
                                            'keep *_hfreco_*_*', 
                                            'keep *_horeco_*_*',
-                                           'keep HBHERecHitsSorted_hbherecoMB_*_*',
-                                           'keep HORecHitsSorted_horecoMB_*_*',
-                                           'keep HFRecHitsSorted_hfrecoMB_*_*',
-                                           'keep ZDCDataFramesSorted_*Digis_*_*',
-                                           'keep ZDCRecHitsSorted_*_*_*',
+                                           #'keep HBHERecHitsSorted_hbherecoMB_*_*',
```

Comment:
let's not drop something that's not in the output that you are using for testing.

We need to get rid of only the "*" in the second field.